wordpress-mobile/WordPress-Android
# Issue/11769 feature announcement dialog

Part of #11769

This PR adds a basic dialog fragment with progress indicator and close button.
For testing purposes, progress indicator is simulated and stays for 3 seconds.

To keep PR size small I'll add an adapter with dummy content in following PR.

[![Image from Gyazo](https://i.gyazo.com/16a66907e7ee63711c12a91fcd6abee2.png)](https://gyazo.com/16a66907e7ee63711c12a91fcd6abee2)

To test:
- Navigate to App Settings and open the What's New dialog from the preference at the bottom.
- Make sure dialog is displayed and that it looks like mocks.
- Click on the CONTINUE button and make sure dialog is closed.

PR submission checklist:
